configs/dmmm.yaml

```diff
@@ -54,3 +54,9 @@ datasets:
     edge_files: ["phosphosite-irefindex13.0-uniprot.txt"]
     other_files: []
     data_dir: "datasets/hiv/processed"
+  - label: dmmmyeast
```

Comment:
Leftover mistake from removing too much from the organization PR: this should still be added to `spras-benchmarking`.